ddtrace/contrib/trace_utils.py

```diff
@@ -650,3 +650,17 @@ def set_user(tracer, user_id, name=None, email=None, scope=None, role=None, sess
             "See https://docs.datadoghq.com/security_platform/application_security/setup_and_configure/"
             "?tab=set_user&code-lang=python for more information.",
         )
+
+
+def extract_info_from_url(url):
```

Comment:
Since this function is in a utils file, you might consider using a name that more explicitly describes what it does (extracting the netloc and querystring from a url).

Reply:
good call - done